main.py

```diff
@@ -0,0 +1,53 @@
+from copy import deepcopy
+
+def primary_forbidden_sequences(list_of_all_forbidden_sequences,P):
+    all_sequences = deepcopy(list_of_all_forbidden_sequences)
+    index_to_be_taken = [0]*P
+    for i in range(P):
+        if index_to_be_taken[i] == 0:
+            current_sequence = all_sequences[i]
+            n = len(current_sequence)
+            working_sequences = deepcopy(all_sequences)
```

Comment:
Why do you use deepcopies?

Reply:
It's just the way I learned to do in prépa when I wanted to be sure my initial list wouldn't change. Would copies be enough ?